bin/dbServer.py

```diff
@@ -31,49 +31,29 @@
 from flask import Flask, request
 import json
 import logging as log
-import os
 import sys
-
-import ConfigParser
-import sqlalchemy
-from sqlalchemy.engine.url import URL
-
 from lsst.dax.dbserv import dbREST_v0
+from lsst.db.engineFactory import getEngineFromFile
 
 app = Flask(__name__)
 
-def initEngine():
-    config = ConfigParser.ConfigParser()
-    defaults_file = os.path.expanduser("~/.lsst/dbAuth-dbServ.txt")
-    config.readfp(open(defaults_file))
-    db_config = dict(config.items("mysql"))
-    # Translate user name
-    db_config["username"] = db_config["user"]
-    del db_config["user"]
-    # SQLAlchemy part
-    url = URL("mysql",**db_config)
-    return sqlalchemy.create_engine(url)
-
-engine = initEngine()
-
+# Configure Engine
+defaults_file = "~/.lsst/dbAuth-dbServ.ini"
+engine = getEngineFromFile(defaults_file)
 app.config["default_engine"] = engine
 
-@app.route('/')
-def getRoot():
-    return '''Test server for testing db. Try adding /db to URI.
-'''
 
-@app.route('/db')
-def getDb():
-    '''Lists supported versions for /db.'''
+@app.route('/')
+def application_root():
```

Comment:
With all of these function name changes (getRoot -> application_root, etc.) I don't see any changes to the client code as part of this ticket.  Are there any users of this service yet?